src/main/scala/beam/sim/RideHailFleetInitializer.scala

```diff
@@ -319,7 +319,7 @@ object RideHailFleetInitializer extends OutputDataDescriptor with LazyLogging {
       val powertrain = new Powertrain(beamVehicleType.primaryFuelConsumptionInJoulePerMeter)
 
       val managerIdDependsOnWhetherVehicleIsCav =
-        if (beamVehicleType.isCav) rideHailManagerId else VehicleManager.AnyManager.managerId
+        if (beamVehicleType.isFullSelfDriving) rideHailManagerId else VehicleManager.AnyManager.managerId
```

Comment:
Should this be cav || fullselfdriving?

Reply:
I have resolved this confusing. As of now, all CAVs have a level of automation >= 4 and are considered by BEAM as robot taxis. No need to separate between FullSelfDriving (level 5) and level 4